scripts/install.py

```diff
@@ -0,0 +1,22 @@
+import launch
+import os
+from modules import shared

```

Comment:
importing `share` in `installer` is a bad idea
`share` includes importing of `gradio`
which may dramatically slow load time

Reply:
:|  
ok can I store "gpu_temps_sleep_temperature_src" value on json file ? 
I want only install  when select "NVIDIA & AMD - openHardwareMonitor"